frontend/src/app/pages/exercises/exercise/shared/exercise-map/utility/feature-manager.ts

```diff
@@ -26,6 +26,11 @@ export interface FeatureManager<ElementFeature extends Feature<any>> {
         feature: ElementFeature
     ) => void;
 
+    /**
+     * @returns whether the feature can be moved by the user
```

Comment:
```suggestion
     * @returns whether the feature can be moved by the user (when having succifient rights to do so)
```

Not sure about that (I know it is only visual and moving objects is controlled by the API), what do you think @Dassderdie ?

Reply:
This would be wrong.

This is a completely independent API from the actions.
E.g. a transferLine or the trash bin at the top of the screen are not even in the state and could be made user moveable by this...

Reply:
Ok, resolving this